shell/platform/windows/compositor_software.cc

```diff
@@ -0,0 +1,54 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#include "flutter/shell/platform/windows/compositor_software.h"
+
+#include "flutter/shell/platform/windows/flutter_windows_view.h"
+
+namespace flutter {
+
+CompositorSoftware::CompositorSoftware(FlutterWindowsEngine* engine)
+    : engine_(engine) {}
+
+bool CompositorSoftware::CreateBackingStore(
+    const FlutterBackingStoreConfig& config,
+    FlutterBackingStore* result) {
+  size_t size = config.size.width * config.size.height * 4;
+  void* allocation = std::calloc(size, sizeof(uint8_t));
+  if (!allocation) {
+    return false;
+  }
+
+  result->type = kFlutterBackingStoreTypeSoftware;
+  result->software.allocation = allocation;
+  result->software.height = config.size.height;
+  result->software.row_bytes = config.size.width * 4;
+  result->software.destruction_callback = nullptr;
```

Comment:
Where is the wiring to call back into `CompositorSoftware::CollectBackingStore`? It might be worth adding a comment here explaining that.

Reply:
Added comments!